src/shared/components/DynamicFunctionsList.tsx

```diff
@@ -60,6 +64,74 @@ const DynamicFunctionsList: FC<Props> = ({onSelect}) => {
     [onSelect]
   )
 
+  if (isFlagEnabled('showFnPath')) {
+    const _search = search.toLocaleLowerCase()
+    const filteredpaths = fluxFunctions
+      .filter(fn =>
+        `${fn.path} ${fn.package} ${fn.name}`
+          .toLocaleLowerCase()
+          .includes(_search)
+      )
+      .reduce((acc, curr) => {
+        if (!acc[curr.path]) {
+          acc[curr.path] = []
+        }
+
+        acc[curr.path].push(curr)
+        return acc
+      }, {})
+    const filtereditems = Object.keys(filteredpaths)
+      .sort((a, b) =>
+        a.toLocaleLowerCase().localeCompare(b.toLocaleLowerCase())
+      )
+      .reduce((acc, curr) => {
+        acc.push(
+          <div className="flux-toolbar--path" key={`fn-head-${curr}`}>
+            {curr}
+          </div>
+        )
+
+        return acc.concat(
```

Comment:
Why you use `push` in line 88 but use `concat` here? Is it because you want to make it clear to return a new set of array?

Reply:
the push is for adding just one item and concat is for adding a bunch of items